### PR TITLE
[IMP]KBT-89: Create Purchase Order API

### PR DESCRIPTION
**Pull request by kongrattapong**

### CODE REVIEW

**Comment by isares-br** [comment]
Pls change PR name into task key

File: addons/kbt_purchase_api/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    'category': 'Company',
+    'website': 'https://roots.tech',
+    'version': '15.0.0.0.1',
+    'license': 'LGPL-3',
+    'contributors': [
+        'Rattapong C.',
+    ],
+    'depends': [
+        'purchase',
+        'kbt_core_update',
```
[the remark this answers is not in the document]
Comment (kongrattapong) [reply addons/kbt_purchase_api/__manifest__.py]: Thanks for reviews, Removed it.

**Comment by isares-br** [comment]
Writing Test too

**Comment by kongrattapong** [comment]
> Writing Test too

Thank you for review, Added

**Review by isares-br — DISMISSED** [review]

**Review by isares-br — APPROVED** [review]

**Pull request merged**